dist/templates/ovn-setup.yaml.j2

```diff
@@ -88,6 +88,11 @@ rules:
   resources:
   - customresourcedefinitions
   verbs: ["list", "get", "watch"]
+- apiGroups:
+  - k8s.ovn.org
+  resources:
+  - dnsobjects
```

Comment:
suggest '*' here

Reply:
the dnsobject type needs much more permissions then the other objects in the k8s.ovn.org apiGroup do you still suggest *?

Reply:
Aha, disregard then.